Dockerfile

```diff
@@ -13,6 +13,14 @@ RUN GEN_DEP_PACKS="software-properties-common \
     language-pack-en-base \
     tmpreaper \
     cron \
+    dnsutils \
```

Comment:
This doesn't list curl anymore -- should it or is curl installed natively?   If not, I think it should be part of the image.

Reply:
I'm showing curl available from inside `apache` in the current 1.1.2.dev build

```Noahs-MacBook-Pro:data noah$  docker ps
CONTAINER ID        IMAGE                                               COMMAND                  CREATED             STATUS              PORTS                                                                                          NAMES
413180a2e487        selenium/node-chrome:3.141.59-neon                  "/opt/bin/entry_poin…"   44 hours ago        Up 44 hours                                                                                                        isle-chrome-ld
f9fc0caa6797        islandoracollabgroup/isle-imageservices:1.1.2.dev   "/init"                  44 hours ago        Up 44 hours         0.0.0.0:8083->8080/tcp                                                                         isle-images-ld
4fbefc716ed8        islandoracollabgroup/isle-apache:1.1.2.dev          "/init"                  44 hours ago        Up 44 hours         80/tcp                                                                                         isle-apache-ld
fd6c951d638e        islandoracollabgroup/isle-fedora:1.1.2.dev          "/init"                  44 hours ago        Up 44 hours         0.0.0.0:8081->8080/tcp                                                                         isle-fedora-ld
f3631b05fd8b        islandoracollabgroup/isle-solr:1.1.2.dev            "/init"                  44 hours ago        Up 44 hours         8983/tcp, 0.0.0.0:8082->8080/tcp                                                               isle-solr-ld
8d4b29b2cce2        selenium/hub:3.141.59-neon                          "/opt/bin/entry_poin…"   44 hours ago        Up 44 hours         0.0.0.0:4444->4444/tcp                                                                         isle-hub-ld
836d2aace2fe        islandoracollabgroup/isle-mysql:1.1.2.dev           "docker-entrypoint.s…"   44 hours ago        Up 44 hours         3306/tcp, 33060/tcp                                                                            isle-mysql-ld
94cd2ca9785a        traefik:1.7.9                                       "/traefik"               44 hours ago        Up 44 hours         0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 0.0.0.0:8080->8080/tcp                               isle-proxy-ld
e5c782a08a86        portainer/portainer                                 "/portainer -H unix:…"   44 hours ago        Up 44 hours         0.0.0.0:9010->9000/tcp                                                                         isle-portainer-ld
88f94d9937b1        traefik:1.6.3-alpine                                "/lando-entrypoint.s…"   45 hours ago        Up 45 hours         0.0.0.0:8888->80/tcp, 0.0.0.0:4433->443/tcp, 0.0.0.0:32771->443/tcp, 0.0.0.0:58087->8080/tcp   landoproxyhyperion5000gandalfedition_proxy_1
Noahs-MacBook-Pro:data noah$ docker exec -it isle-apache-ld bash
root@4fbefc716ed8:/# ls
bin  boot  dev  etc  home  init  lib  lib64  libexec  media  mnt  opt  proc  root  run  sbin  srv  sys  tmp  usr  utility-scripts  var
root@4fbefc716ed8:/# curl google.com
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="http://www.google.com/">here</A>.
</BODY></HTML>```